shared/src/main/scala/drt/shared/PortState.scala

```diff
@@ -194,7 +195,21 @@ class PortStateMutable {
     PortState(flights = fs, crunchMinutes = cms, staffMinutes = sms)
   }
 
-  def flightsRange(roundedStart: SDateLike, roundedEnd: SDateLike): ISortedMap[UniqueArrival, ApiFlightWithSplits] = flights.range(roundedStart, roundedEnd)
+  def flightsRange(start: SDateLike, end: SDateLike): ISortedMap[UniqueArrival, ApiFlightWithSplits] = {
```

Comment:
check if this function is still used, or should be replaced with the above one.

Reply:
Reverted this change as it's indeed covered by the `flights.range()` function it calls